documentation/Using-the-SDK/Building-a-table.md

```diff
@@ -1,21 +1,27 @@
 # Building a Table
 
-Table construction happens by interacting with an instance of an `ITableBuilder`. Broadly, `Tables` consist of `TableConfigurations` and Columns. Columns can be further broken down into `ColumnConfigurations` and `Projections`. An `ITableBuilder` has methods to add both Columns and `TableConfigurations`.
+Table construction happens by interacting with an instance of an `ITableBuilder`. Broadly, `Tables` consist of `TableConfigurations` and Columns. Columns can be further broken down into `ColumnConfigurations` and `Projections`. An `ITableBuilder` has methods to add `TableConfigurations` and to establish a row count for the table.
```

Comment:
should we keep "both Columns and"?

Reply:
Wonder if this should be a list

"An `ITableBuilder has methods to
1) Add table configurations
2) Set a row count
3) Add columns

Reply:
no, you cannot use it to add columns.

Reply:
to be clear, there is no ITableBuilder.AddColumn method